modules/ROOT/attachments/specs/enotices2-ted-europa-eu-esenders.yaml

```diff
@@ -12,47 +12,87 @@ servers:
 security:
   - esenders-token: []
 paths:
-  /api/notice/notices/{businessId}/validation-report:
+  /api/notice/notices/{noticeId}/validation-report:
     get:
       tags:
         - Notice
       summary: Get notice validation report
+      description: Get validation report created during notice submission request
       operationId: getValidationReport
       parameters:
-        - name: businessId
+        - name: noticeId
+          description: Notice identifier
           in: path
           required: true
           schema:
             type: string
             format: uuid
       responses:
         '200':
-          description: OK
+          description: The report is returned.
           content:
             '*/*':
               schema:
                 type: string
                 format: binary
-  /api/notice/notices/{businessId}/stop-publication-esentool:
+        '404': 
+          description: |-
+            Possible issues:
+            - Notice with given noticeId not found
+            - Report for given noticeId not found (it can be that is temporarily not available, please re-try later)
+          content:
+            application/json: {}
+        '500':
+          description: |-
+            Possible issues:
+            - TED API key is missing or is not valid.
+            - No Enotices2 account found for the EU login account identified by provided TED API key.
+            - Several versions of notice with given noticeId exist.
+          content:
+            application/json: {}
+         
+  /api/notice/notices/{noticeId}/stop-publication-esentool:
     post:
       tags:
         - Notice
-      summary: send stop notice publication to ted monitor
+      summary: Request to stop the publication of a notice
       operationId: stopPublicationEsentool
       parameters:
-        - name: businessId
+        - name: noticeId
+          description: Notice identifier
           in: path
           required: true
           schema:
             type: string
             format: uuid
       responses:
         '200':
-          description: OK
+          description: Notice publication stopped
```

Comment:
Please change to "Notice publication stopped successfully", to make it clear that the request was successful.

Reply:
I agree with @meletev here. Description of status 200 should always be clear that the request was successful. Changing it rom "OK" to "blah blah stopped" is confusing.